### PR TITLE
Add validate import file, and fix issues about import

### PR DESCRIPTION
**Pull request by patrycja-dev41171**
#54 Added validation to imported file from FE.

### CODE REVIEW

File: routers/upload-file.ts [hunk 1]
```diff
@@ -8,15 +8,25 @@ uploadRouter.post('/', async (req, res) => {
     throw new ValidationError('Nie przesłano pliku !');
   }
 
-  try {
-    const [data] = Object.entries(req.files);
-    const primaryData: any = data[1];
+  const [data] = Object.entries(req.files);
```
Comment (patrycja-dev41171) [inline routers/upload-file.ts:11]: Code styling?

File: routers/upload-file.ts [hunk 2]
```diff
@@ -8,15 +8,25 @@ uploadRouter.post('/', async (req, res) => {
...
-    throw new ValidationError('Błąd spróbuj ponownie później');
+  if (primaryData.mimetype !== 'application/json') {
+    throw new ValidationError('Plik musi być w formacie JSON !');
   }
+
+  const parseData = JSON.parse(primaryData.data);
+  const emailValidate = new RegExp('@');
+
+  const validateData = parseData.map((el: any) => {
+    if (!emailValidate.test(el.email)) {
```
Comment (patrycja-dev41171) [inline routers/upload-file.ts:22]: Good understanding of validation.

**Review by patrycja-dev41171 — COMMENTED** [review]
Good Code.

**Comment by mae1579** [comment]
I understand :)

**Pull request merged**